### ISSUE TITLE
Custom vector visualizer not working

### ISSUE THREAD

**Issue by thecommonegret** [mapbox/mapbox-unity-sdk#771]
Hi,
I am trying to utilize NYC's street tree dataset (Mapbox tileset uploaded [here](https://www.mapbox.com/studio/tilesets/bwellington.3r3ygr95/)) to visualize in a Manual Alignment AR scene, but can't seem to get them to appear. I'm currently just using the diameter of the tree (property: _tree_dbh_) as the extrusion value (which I know doesn't make much sense, but just trying to get something to show up). One thing I noticed is that all properties of the tileset are being read as strings even though some are just numbers. Maybe this is the issue? I uploaded into Mapbox as a geojson file. Ideally I'd eventually like to have these street trees appear as green, yellow, and red 'cylinders' according to their recorded health, as styled [here](https://api.mapbox.com/styles/v1/bwellington/cjh4wdm6o2kze2sqqca0b99ls.html?fresh=true&title=true&access_token=pk.eyJ1IjoiYndlbGxpbmd0b24iLCJhIjoiY2loNTNjazlwMTA0ZHc5bTU3cmJ3N24zMyJ9.PRgQmgfSQma6GPYEtykZ8Q#15.8/40.797011/-73.947655/0)  

tileset in Mapbox:
![image](https://user-images.githubusercontent.com/39036160/40001837-1a9caeec-575d-11e8-9060-a1a2a78f39cd.png)

setup in Unity:
![image](https://user-images.githubusercontent.com/39036160/40001305-f51b2ae6-575b-11e8-965b-ed1af016473c.png)

Running latest Unity and Mapbox SDK versions.
Will eventually build out to mobile. 

Thank you,
Matthew

_Note: If this is a bug or support ticket, please provide the following information:_
* Unity version: 
  * Scripting Runtime Version: 
  * Scripting Backend: 
  * Api Compatibility Level: 
* Mapbox SDK version: 
* The platform you're building to:
* A description of what you're trying to do:
* Steps to recreate the bug if appropriate:
* Links to your logs are also helpful:

**Comment by willychang21** [comment]
@jordy-isaac will 1.4.2 sdk update today??

**Comment by jordy-isaac** [comment]
@willychang21 Yes, 1.4.2 did release and it has some great POI feature additions. You should check it out. 

As for your ticket. When you run the game are the game objects from your visualizer appearing in your hierarchy and just not being rendered in the scene/game views?

The data type being a string could be the issue as the `height` under [`Extrusion Type`](https://www.mapbox.com/unity-sdk/overview/data-explorer/#buildings) comes from the [Mapbox Streets tileset](https://www.mapbox.com/studio/tilesets/mapbox.mapbox-streets-v7/) and is an number.

**Comment by a100447739** [comment]
@jordy-isaac 
I opened the file sdkproject in mapbox-unity-sdk-develop
but there's some error
![image](https://user-images.githubusercontent.com/37835383/40058147-da4ed5c8-5882-11e8-92d6-a3fa4d8a6fbb.png)
![image](https://user-images.githubusercontent.com/37835383/40058179-ee1f830e-5882-11e8-8f6a-016164bb1a35.png)
how to fix it??

**Comment by thecommonegret** [comment]
@jordy-isaac 
The game objects do appear to be showing up in the hierarchy when I run the game. See below:

![image](https://user-images.githubusercontent.com/39036160/40060012-8df048e2-5823-11e8-92e4-b0f675a14b79.png)

**Comment by willychang21** [comment]
@jordy-isaac 
Thanks,i figure it out!!

**Comment by atripathi-mb** [comment]
@thecommonegret In the screenshot your tileset contains only points, but in unity you have selected `primitive type` as `line` . Line mesh generation needs minimum two vertices. Could you change primitive type to point and use a prefab modifier to place cylinders at feature locations and try?

**Comment by atripathi-mb** [comment]
@a100447739 you are missing dependencies in your project. Please run the `update-mapbox-unity-sdk-core` (bat or sh depending on your platform) script to pull all the dependencies.

**Comment by BergWerkGIS** [comment]
@a100447739 

> you are missing dependencies in your project

See ReadMe:
https://github.com/mapbox/mapbox-unity-sdk#from-this-repository-advanced-latest-development
